### PR TITLE
add data and script to load state, districts data

### PR DESCRIPTION
**Pull request by anroopak**
Adds the State-District JSON and a management command to load the data.

### CODE REVIEW

File: care/users/management/commands/load_state_data.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        json_file_path = options["json_file_path"]
+
+        data = []
+        with open(json_file_path, "r") as json_file:
+            data = json.load(json_file)
+
+        for item in data:
+            state_name = item["state"].strip()
+            if state_name.lower() in states_to_ignore:
+                print(f"Skipping {state_name}")
```
Comment (afzalIbnSH) [inline care/users/management/commands/load_state_data.py:34]: We don't have a logger setup ?

File: care/users/management/commands/load_state_data.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                continue
+
+            districts = [d.strip() for d in item["districts"].split(",")]
+
+            state, is_created = State.objects.get_or_create(name__iexact=state_name, defaults={"name": state_name})
+            print(f"{state_name} - {is_created}")
```
Comment (afzalIbnSH) [inline care/users/management/commands/load_state_data.py:40]: Something like `f"{'Created' if is_created else 'Retrieved'} {state_name}"` could potentially be more helpful. It'll log "Created Kerala" for instance.

File: care/users/management/commands/load_state_data.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+            for d in districts:
+                _, is_created = District.objects.get_or_create(state=state, name__iexact=d, defaults={"name": d})
+                print(f"{d} - {is_created}")
```
Comment (afzalIbnSH) [inline care/users/management/commands/load_state_data.py:44]: Same as above

File: setup.cfg [hunk 4]
```diff
@@ -26,4 +26,4 @@ django_settings_module = config.settings.test
 ignore_errors = True
 
 [isort]
-known_third_party = allauth,celery,crispy_forms,dateparser,dateutil,django,django_filters,django_rest_passwordreset,djangoql,djqscsv,drf_extra_fields,drf_yasg,dry_rest_permissions,environ,fernet_fields,freezegun,hardcopy,location_field,multiselectfield,partial_index,phonenumber_field,phonenumbers,pytz,ratelimit,requests,rest_framework,rest_framework_nested,rest_framework_simplejwt,sentry_sdk,simple_history
+known_third_party = allauth,boto3,celery,crispy_forms,dateparser,dateutil,django,django_filters,django_rest_passwordreset,djangoql,djqscsv,drf_extra_fields,drf_yasg,dry_rest_permissions,environ,fernet_fields,freezegun,hardcopy,location_field,multiselectfield,partial_index,phonenumber_field,phonenumbers,pytz,pywebpush,ratelimit,requests,rest_framework,rest_framework_nested,rest_framework_simplejwt,sentry_sdk,simple_history
```
Comment (afzalIbnSH) [inline setup.cfg:29]: Looks like this PR adds `boto3 ` and `pywebpush` but it's not clear why.

Comment (vigneshhari) [reply setup.cfg:29]: we use both these libraries now

File: care/users/management/commands/load_state_data.py (hunk 1 above)
Comment (vigneshhari) [reply care/users/management/commands/load_state_data.py:34]: We don't store any kind of logs as of now, We can start that if needed.

File: care/users/management/commands/load_state_data.py (hunk 2 above)
Comment (vigneshhari) [reply care/users/management/commands/load_state_data.py:40]: Will Change it to that

File: care/users/management/commands/load_state_data.py (hunk 3 above)
Comment (vigneshhari) [reply care/users/management/commands/load_state_data.py:44]: Will Change it to that

**Pull request merged**